src/Components/FilterForm/FilterForm.tsx

```diff
@@ -76,6 +77,8 @@ export default function FilterForm() {
   const [salaryMax, setSalaryMax] = useState<number>(0);
   const [internshipPreference, setInternshipPreference] = useState<number>(0);
   const [experience, setExperience] = useState<number>(0);
+  const [filterUrl, setFilterUrl] = useState<string>("");
+  const filterData = `${courseEvaluation}/${activityEvaluation}/${codeEvaluation}/${teamEvaluation}/${workplacePreference}/${contractPreference}/${salaryMin}/${salaryMax}/${internshipPreference}/${experience}`;
```

Comment:
it looks like the form query is hardcoded and is only created on the first render. This should be defined in the function on click. Makes the link real value.